content/en/docs/concepts/configuration/pod-overhead.md

```diff
@@ -0,0 +1,57 @@
+---
+reviewers:
+- dchen1107
+- egernst
+- tallclair
+title: Pod Overhead
+content_template: templates/concept
+weight: 20
+---
+
+{{% capture overview %}}
+
+{{< feature-state for_k8s_version="v1.16" state="alpha" >}}
+
+
+When you run a Pod on a Node, the Pod itself takes an amount of system resources. These
+resources are additional to the resources needed to run the container(s) inside the Pod.
+_Pod Overhead_ is a feature for accounting for the resources consumed by the pod infrastructure
+on top of the container requests & limits.
+
+
+{{% /capture %}}
+
+
+{{% capture body %}}
+
+## Pod Overhead
+
+In Kubernetes, the pod's overhead is set at
+[admission](https://kubernetes.io/docs/reference/access-authn-authz/extensible-admission-controllers/#what-are-admission-webhooks)
+time according to the overhead associated with the pod's
+[RuntimeClass](https://kubernetes.io/docs/concepts/containers/runtime-class/).
+
+### Set Up
+
+You need to make sure that the `PodOverhead`
+[feature gate](/docs/reference/command-line-tools-reference/feature-gates/) is enabled (it is off by default)
+across your cluster. This means:
+- in {{< glossary_tooltip text="kube-scheduler" term_id="kube-scheduler" >}}
+- in {{< glossary_tooltip text="kube-apiserver" term_id="kube-apiserver" >}}
+- in the {{< glossary_tooltip text="kubelet" term_id="kubelet" >}} on each Node
+- in any custom API servers that use feature gates
+
+{{< note >}}
+Users who can write to RuntimeClass resources are able to have cluster-wide impact on
+workload performance. You can limit access to this ability using Kubernetes access controls.
+See [Authorization Overview](/docs/reference/access-authn-authz/authorization/) for more details.
+{{< /note >}}
+
```

Comment:
Does the scheduler make different decisions based on the PodOverhead?

What kind of impact does this overhead have, if I configure it?

Reply:
Indeed, it does impact scheduling, as well as some kubelet behavior.  I added a short blurb on this, and it is discussed in detail in the design doc (KEP).  I plan to write a more thorough overview of the feature and example usage in a blog post, though I'm not sure if it makes sense here?

Reply:
That makes sense.

_Optional tweak_
How about adding a link from https://kubernetes.io/docs/concepts/scheduling/kube-scheduler/#what-s-next to the new page on Pod Overhead?

Reply:
Sure. done.